enhancements/cluster-logging/loki-storage.md

```diff
@@ -0,0 +1,605 @@
+
+* Support a limited set of Loki scale outs, which are battle-tested on our internally managed Loki deployment. The target scale outs are defined using the format `Nx.{extra-small, small, medium}` where `N` the number of Loki deployments:
+  * **1x.extra-small**: Dedicated scale for development and demo purposes only! Not a valid configuration for cluster logging production environments.
+  * **1x.small**: Dedicated scale for low-retention log storage without replication factor support. All components are deployed with HA by running at least two replicas.
+  * **1x.medium**: Dedicated scale for log-retention log storage with single replication factor support. All components are deployed with HA by running at least two replicas. The only exception is the ingester that requires at minimum three replicas to allow failover when single replication factor enabled.
+
+* Support tuning of CPU/Memory resource and limits for each individual component beyond the selected scale-out size.
+
+* Support adding kubernetes tolerations and node selectors for all Loki components individually. This enables operating the Loki deployment on dedicated nodes.
+
+* Support a limited set of control options from Loki's [limit_config](https://grafana.com/docs/loki/latest/configuration/#limits_config) for the ingestion and query paths. The set is an opinionated list of options to enable tuning of object storage usage. The list of options is offered globally across the whole Loki deployment or per tenant.
+
+For **1x.medium** instances only:
+
+* Support optionally replication factor of two for log store on the target object storage.
+
+* Support auto-compaction of the index and chunks stored on the object storage when replication factor of two enabled.
+
+### User Stories
+
+* As a cluster logging admin, I want to be able to configure Loki as an alternative log store in the `ClusterLogging` custom resource.
+
+* As a cluster logging admin, I want to be able to split logs to Loki as an alterantive log store in the `ClusterLogForwarder` custom resources to be able to support custom tenants.
+
+* As a cluster logging admin, I want to select a different scale for my Loki deployment to enable single replication factor and auto-compaction.
+
+* As a cluster logging admin, I want to expand resources/limits for individual Loki components to support load scenarios not declared by the cluster t-shirt size selected.
+
+* As a cluster logging admin, I want to configure scheduling of individual Loki components to custom selected nodes on my cluster to separate resource usage of the cluster logging from other workloads on my cluster.
+
+* As a Cluster Logging admin, I want to tune the ingestion limits globally or per tenant to support custom ingestion characteristics (e.g. ingenstion rate, label cardinality).
+
+* As a Cluster Logging admin, I want to tune the query limits globally or per tenant to support custom query characteristics (e.g. chunks per series returned).
```

Comment:
to go along with my prior comment we could update this to something along the lines of "As a Loki Operator consumer"